src/http/routes/product.js

```diff
@@ -55,21 +65,25 @@ async function update(req, res) {
       },
     })
     res.send(200)
+    return next()
   } else {
-    res.send(400, {
-      message: `Bad request: param ID (${id}) does not match body ID (${req.body.id}).`,
-    })
+    return next(
+      new BadRequestError(
+        `Bad request: param ID (${id}) does not match body ID (${req.body.id}).`
+      )
+    )
   }
 }
 
-async function remove(req, res) {
+async function remove(req, res, next) {
   const id = getIdParam(req)
   await models.Product.destroy({
     where: {
       id: id,
     },
   })
   res.send(200)
```

Comment:
Status code 204 (No Content) is preferable when nothing is returned by the server